biodata-formats/src/main/java/org/opencb/biodata/formats/variant/VariantAnnotationUtils.java

```diff
@@ -0,0 +1,689 @@
+            return null;
+        }
+        StringBuilder stringBuilder = new StringBuilder(aa);
+
+        for (int i = 1; i < stringBuilder.length(); i++) {
+            stringBuilder.setCharAt(i, String.valueOf(stringBuilder.charAt(i)).toLowerCase().charAt(0));
+        }
+
+        return stringBuilder.toString();
+    }
+
+    public static Boolean isSynonymousCodon(String codon1, String codon2) {
+        return isSynonymousCodon(false, codon1, codon2);
+    }
+
+    public static Boolean isSynonymousCodon(Boolean mitochondrialCode, String codon1, String codon2) {
+//        Map<String, String> geneticCode = null;
+        if (mitochondrialCode) {
+            return MT_IS_SYNONYMOUS_CODON.get(codon1.toUpperCase()).get(codon2.toUpperCase());
+        } else {
+            return IS_SYNONYMOUS_CODON.get(codon1.toUpperCase()).get(codon2.toUpperCase());
+        }
+    }
+
+    public static Boolean isStopCodon(String codon) {
+        return isStopCodon(false, codon);
+    }
+
+    public static Boolean isStopCodon(boolean mitochondrialCode, String codon) {
+        if (mitochondrialCode) {
+            if (codon.equals(TAA) || codon.equals(TAG) || codon.equals(AGA) || codon.equals(AGG)) {
+                return true;
+            }
+        } else {
+            if (codon.equals(TAA) || codon.equals(TGA) || codon.equals(TAG)) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    public static boolean isStartCodon(boolean mitochondrialCode, String codon) {
+        if (mitochondrialCode) {
+            if (codon.equals(ATG) || codon.equals(ATA)) {
+                return true;
+            }
+        } else {
+            if (codon.equals(ATG)) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+
+    public static String getAminoacid(boolean mitochondrialCode, String codon) {
+        if (mitochondrialCode) {
+            return MT_CODON_TO_A.get(codon);
+        } else {
+            return CODON_TO_A.get(codon);
+        }
+    }
+
+    public static List<SequenceOntologyTerm> getSequenceOntologyTerms(Iterable<String> soNames) throws SOTermNotAvailableException {
+        List<SequenceOntologyTerm> sequenceOntologyTerms = new ArrayList<>();
+        for (String name : soNames) {
+            name = fixSONameIfNeeded(name);
+            sequenceOntologyTerms.add(newSequenceOntologyTerm(name));
+        }
+        return sequenceOntologyTerms;
+    }
+
+    private static String fixSONameIfNeeded(String name) {
+        String fixedName = SO_NAMES_CORRECTIONS.get(name);
+        return fixedName == null ? name : fixedName;
+    }
+
+    public static SequenceOntologyTerm newSequenceOntologyTerm(String name) throws SOTermNotAvailableException {
+        return new SequenceOntologyTerm(ConsequenceTypeMappings.getSoAccessionString(name), name);
+    }
+
+    public static String buildVariantId(String chromosome, int start, String reference, String alternate) {
```

Comment:
Unused method. This method is intended to be used to create an `_id` for mongodb. It should not be here

Reply:
This includes `appendChromosome(..)` and `SEPARATOR_CHAR`